### ISSUE TITLE
Allow gestures to be disabled

### ISSUE THREAD

**Issue by ghost** [adrcotfas/goodtime#180]
It's very easy for me to accidentally swipe my screen and reset my timer. Combined with the fact that I cannot jump my timer to a specific time, I lose my current work session and have to start from the beginning.

This could be solved with either:
1. Adding a setting to disable gestures, or
2. Adding a setting to confirm "destructive" actions such as resetting the session.

**Comment by colinner** [comment]
I agree. In addition, I feel that the gestures are unintuitive, and I forget what they do after not using the app for awhile. I think that a set of standard play/pause/fastForward/skip buttons would be more intuitive.

**Comment by Erudition** [comment]
Same here, too easy to accidentally swipe, and sometimes you don't even notice that you lost your place and the timer reset, since it keeps running.

**Comment by fcr--** [comment]
I also agree here, gestures are very inintuitive and destructive, for example: if swipe up adds a minute, wouldn't it make sense for swipe down to reduce a minute?  This behavior means that if you accidently make a gesture, even if you realize the mistake, it's very confusing how to undo it.

Since swipe left/right is also destructive, so the best alternative would probably be to disable them.

**Comment by adrcotfas** [comment]
The next major update will feature a nicer UX for gestures and I will consider adding buttons as an alternative.
